spec/controllers/admin/reports/registered_adviser_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe Admin::Reports::RegisteredAdviserController, type: :controller do
+  describe '#filename' do
```

Comment:
@margOnline the filename is an option that is merely passed to the render object by the controller though. Since the filename is first mentioned in the controller, I believe this is the best place to test it. wdyt?